DOL.WHD.Section14c.Test/ActionFilters/GlobalExceptionAttributeTests.cs

```diff
@@ -72,72 +72,94 @@ public void GlobalExceptionAttribute_AllowMultipleTest()
 
 
         [TestMethod()]
-        [ExpectedException(typeof(HttpResponseException),
-       "Internal Server Error.")]
         public void GlobalExceptionAttribute_OnExceptionTest_ValidationException()
         {
-            httpActionExecutedContext.Exception = new ValidationException();
-            filter.OnException(httpActionExecutedContext);
+            try
+            {
+                httpActionExecutedContext.Exception = new ValidationException();
+                filter.OnException(httpActionExecutedContext);
+            }
+            catch(HttpResponseException ex)
+            {
+                Assert.AreEqual(ex.Response.StatusCode, HttpStatusCode.BadRequest);
+            }
```

Comment:
I like this better - it still doesn't do what I was hoping for, but as you explained yesterday, that's because of the way the tracer gets instantiated (it's the same issue as in `LoggingFilterAttribute_OnActionExecutingTest`).  Thanks for bearing with me as we chased that down!